marklogic-data-hub-central/build-rpm.gradle

```diff
@@ -0,0 +1,55 @@
+ospackage {
+    packageName = 'hub-central'
+    version = "${project.version}"
+    release = 1
+    os = LINUX
+    type = BINARY
+    arch = NOARCH
+    summary = 'Hub Central Application Server'
+    url = 'http://marklogic.com/'
+    vendor = 'MarkLogic Corporation'
+
+    preInstall file('src/rpm/scripts/pre-install.sh')
+    postInstall file('src/rpm/scripts/post-install.sh')
+    preUninstall file('src/rpm/scripts/pre-uninstall.sh')
+    postUninstall file('src/rpm/scripts/post-uninstall.sh')
+
+    requires('systemd')
+
+    user 'hc-runner'
+    permissionGroup 'hc-runner'
+
+    from(war.outputs.files) {
+        rename { String fileName ->
+            fileName.replace("marklogic-data-hub-central-${project.version}", "hub-central")
+        }
+        fileMode 0755
+        into '/opt/hub-central'
+    }
+
+    from('src/rpm/services/hub-central.service') {
```

Comment:
This is really cool, thanks for getting this together. Makes me think we should opt for an rpm whenever possible now.